client/service-container-chain/src/service.rs

```diff
@@ -499,7 +507,7 @@ fn start_consensus_container(
                     );
 
                     let slot_freq = tc_consensus::min_slot_freq::<Block, ParachainClient, NimbusPair>(
-                        orchestrator_client_for_cidp.as_ref(),
+                        orchestrator_client_for_cidp.as_ref().unwrap(),
```

Comment:
Is it possible to remove the `unwrap` here?

Reply:
*fixed*